docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,46 @@
+.. highlight:: psql
+
+.. sql-start-transaction:
+
+=====================
+``START TRANSACTION``
+=====================
+
+This is PostgreSQL compatibility feature that the statement is implemented but
+will be silently ignored.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Synopsis
+========
+
+::
+
+   START TRANSACTION [ transaction_mode [ , ...] ]
+
+Where ``transaction_mode`` is one of::
+
+   ISOLATION LEVEL isolation_level | (READ WRITE | READ ONLY) | [NOT] DEFERRABLE
+
+Where ``isolation_level`` is one of::
+
+   { SERIALIZABLE | REPEATABLE READ | READ COMMITTED | READ UNCOMMITTED }
+
+
+Description
+===========
+
+CrateDB does not support transactions and will silently ignore the ``START
+TRANSACTION`` statement.
+
+For further details, see :ref:`crate_standard_sql`.
```

Comment:
```suggestion
CrateDB will silently ignore the ``START TRANSACTION`` statement.
```

because we moved this info to the top where readers will see it first, we can afford to trim this bit down by removing any explanation or justification